app/models/vacancy.rb

```diff
@@ -33,13 +33,18 @@ class Vacancy < ApplicationRecord
 
   include AlgoliaSearch
 
+  # rubocop:disable Metrics/BlockLength
   algoliasearch disable_indexing: !Rails.env.production? do
-    attributes :first_supporting_subject, :job_roles, :job_title, :salary, :second_supporting_subject, :working_patterns
```

Comment:
I don't actually see `objectID` anywhere in here?

Reply:
<img width="534" alt="Screenshot 2020-04-30 at 13 42 41" src="https://user-images.githubusercontent.com/20921/80711578-7f3d7000-8ae8-11ea-80f9-38a55c24885a.png">
Are you talking about this?

Reply:
https://www.algolia.com/doc/api-client/methods/indexing/#the-object-id

Reply:
That is a good objectID to have. I want to get rid of extraneous objectIDs.

We currently get an objectID for each Vacancy record and that's fine and should stay that way. Algolia seems to be setting it to `Vacancy.id`.

Example record:

```
{
      "first_supporting_subject": {
        "id": "45eb4f2d-67fd-49d2-b039-8f9b746b75ff",
        "name": "Maths"
      },
      "job_roles": [
        "Suitable for NQTs"
      ],
      "job_title": "Mathematics Teacher ",
      "salary": "Main pay range 1 to Upper pay range 3, £25,543 to £41,635",
      "second_supporting_subject": null,
      "working_patterns": [
        "full_time",
        "part_time"
      ],
      "expiry_date": 1587726000,
      "last_updated_at": 1586224842,
      "listing_status": "published",
      "newly_qualified_teacher_status": true,
      "permalink": "mathematics-teacher-becket-keys-church-of-england-free-school",
      "publication_date": 1585911600,
      "school": {
        "name": "Becket Keys Church of England Free School",
        "address": "Sawyers Hall Lane",
        "county": "Essex",
        "local_authority": "Essex",
        "phase": "secondary",
        "postcode": "CM15 9DA",
        "region": "East of England",
        "town": "Brentwood"
      },
      "start_date": 1598958000,
      "subject": "Maths",
      "_geoloc": {
        "lat": 51.6271124337987,
        "lng": 0.302248497586538
      },
      "objectID": "fff7b992-3066-4818-87d1-9e03fadda4e0"
```

I want to make the supporting subject attributes consistent with `subject` in that they should be a string, not a json containing a name and an objectID.

Reply:
Just realised my confusion: I have been calling `id`s `ObjectID`s in my description and comment.